dbt/adapters/redshift/connections.py

```diff
@@ -496,8 +492,18 @@ def add_query(self, sql, auto_begin=True, bindings=None, abridge_sql_log=False):
             if without_comments == "":
                 continue
 
+            retryable_exceptions = (
+                redshift_connector.InterfaceError,
+                redshift_connector.InternalError,
+            )
+
             connection, cursor = super().add_query(
-                query, auto_begin, bindings=bindings, abridge_sql_log=abridge_sql_log
+                query,
+                auto_begin,
+                bindings=bindings,
+                abridge_sql_log=abridge_sql_log,
+                retryable_exceptions=retryable_exceptions,
```

Comment:
Passing these additional arguments will require a new lower bound on `dbt-adapters`. We don't allow for `**kwargs` in `add_query` (or any of our base methods), so passing a new kwarg will break it.